### PR TITLE
Feature/quest

### PR DESCRIPTION
**Pull request by RickvanBerlo**
This Pull Request (PR) adds functionality to the game according to User Story #XXX

This PR...

* [x] has added functionality.
* [x] complies with the code style of the repository.
* [x] is up to date with the `dev` branch.

Also, I have...

* [x] Updated Taiga.

Please merge with care!

### CODE REVIEW

File: memecity.game/Game/States/PauseMenuState.cpp [hunk 1]
```diff
@@ -7,9 +7,22 @@
 PauseMenuState::PauseMenuState(memecity::engine::state::StateManager & sm, GameManager::GameContext & gc, memecity::engine::ecs::EntityManager & em)
 	: State(sm), _context(&gc), _entity_manager(&em)
 {
+	help_menu4 = memecity::engine::ui::menu::MenuBuilder(gc.get_multimedia_manager())
+		.create_menu("Help", assets::fonts::DEFAULT_FONT)
+		.with_read_only_menu_item("Stats")
+		.with_read_only_menu_item("Strenght: improves attack")
```
Comment (Thomvdpas) [inline memecity.game/Game/States/PauseMenuState.cpp]: Strength

File: memecity.game/Game/States/PauseMenuState.cpp [hunk 2]
```diff
@@ -7,9 +7,22 @@
...
+		.with_read_only_menu_item("Perception: improves vision")
+		.with_read_only_menu_item("Endurance: improves movment speed")
```
Comment (Thomvdpas) [inline memecity.game/Game/States/PauseMenuState.cpp]: Movement

File: memecity.game/Game/States/PauseMenuState.cpp [hunk 3]
```diff
@@ -7,9 +7,22 @@
...
+		.with_read_only_menu_item("Charisma: better change of a getting away with a faulty pickpocket ")
```
Comment (Thomvdpas) [inline memecity.game/Game/States/PauseMenuState.cpp:19]: Chance

File: memecity.game/Game/States/PauseMenuState.cpp [hunk 4]
```diff
@@ -7,9 +7,22 @@
...
+		.with_read_only_menu_item("Intelligence: imporves pickpocket")
```
Comment (Thomvdpas) [inline memecity.game/Game/States/PauseMenuState.cpp]: Improves

File: memecity.game/Game/States/PauseMenuState.cpp [hunk 5]
```diff
@@ -7,9 +7,22 @@
...
+		.with_read_only_menu_item("Agility: imporves defence")
```
Comment (Thomvdpas) [inline memecity.game/Game/States/PauseMenuState.cpp]: Improves defense

File: memecity.game/Game/Builder/QuestBuilder.cpp [hunk 6]
```diff
@@ -1,26 +1,264 @@
...
+			break;
+		}
+	}
+
+	std::deque<Task> tasks_quest1{
+		Task{
+		"Talking to " + appie,
+		Quest_State::Interaction,
+		{"heyy.",
+		"I drunk HAHAHA",
```
Comment (Thomvdpas) [inline memecity.game/Game/Builder/QuestBuilder.cpp:154]: I'm

**Review by Thomvdpas — CHANGES_REQUESTED** [review]

File: memecity.game/Game/Builder/QuestBuilder.cpp (hunk 6 above)
Comment (RickvanBerlo) [reply memecity.game/Game/Builder/QuestBuilder.cpp:154]: was meer een refrence dat hij het niet meer goed kan uitspreken

**Review by Thomvdpas — APPROVED** [review]

**Pull request merged**